gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/EnforceFlowFinishDeadlineDagProc.java

```diff
@@ -60,7 +60,9 @@ protected void enforceDeadline(DagManagementStateStore dagManagementStateStore,
         DagProcUtils.cancelDagNode(dagNodeToCancel, dagManagementStateStore);
       }
 
-      dag.setFlowEvent(TimingEvent.FlowTimings.FLOW_RUN_DEADLINE_EXCEEDED);
+      DagProcUtils.setAndEmitFlowEvent(eventSubmitter, dag, TimingEvent.FlowTimings.FLOW_CANCELLED);
+
+      log.warn("Killing dag {} due to exceeding SLA of {} ms", getDagId(), flowFinishDeadline);
```

Comment:
I like this logging, but maybe let's fold it into:
```
log.info("Found {} DagNodes to cancel (DagId {}).", dagNodesToCancel.size(), getDagId());
```
that happens before the attempts to cancel (in case any fail).  `info` level is probably sufficient, with `warn` too severe

(also: SLA => deadline)